flutter/scripts/generate-cocoa-bindings.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cocoa_version="${1:-$(./scripts/update-cocoa.sh get-version)}"
+
+cd "$(dirname "$0")/../"
+
+# Remove dependency on script exit (even in case of an error).
+trap "dart pub remove ffigen" EXIT
+
+# Currently we add the dependency only when the code needs to be generated because it depends
+# on Dart SDK 3.2.0 which isn't available on with Flutter stable yet.
+# Leaving the dependency in pubspec would block all contributors.
+# As for why this is coming from a fork - because we need a specific version of ffigen including PR 607 but not PR 601
+# which starts generating code not compatible with Dart SDK 2.17. The problem is they were merged in the wrong order...
+dart pub add 'dev:ffigen:{"git":{"url":"https://github.com/vaind/ffigen","ref":"4d714b0f2d11e9698a1885ec2d5de15f2ed827f1"}}'
```

Comment:
Should we fork this under `getsentry`? Or use straight up the `dart-lang` one?

```suggestion
dart pub add 'dev:ffigen:{"git":{"url":"https://github.com/getsentry/ffigen","ref":"4d714b0f2d11e9698a1885ec2d5de15f2ed827f1"}}'
```

Reply:
Create a fork: https://github.com/getsentry/ffigen

@vaind will create a new branch with the sha we need, and we'll point to it from here

Reply:
done